src/Protocol/ForgeMods.cpp

```diff
@@ -0,0 +1,89 @@
+
+// ForgeMods.cpp
+
+// Data structure listing the Forge mods name and versions
+
+#include "Globals.h"
+#include "ForgeMods.h"
+
+cForgeMods::cForgeMods()
+{
+}
+
+
+
+
+
+cForgeMods::cForgeMods(AStringMap a):m_Mods(a)
+{
+	for (const auto & item: a)
+	{
+		m_ModNames.push_back(item.first);
+	}
+}
```

Comment:
```c++
	for (const auto & item: a)
	{
		m_ModNames.push_back(item.first);
	}
```

@peterbell10 I think I understand the problem, but having some trouble fixing it. How exactly could I change it to take a const reference? Is the const reference not `const auto &` in the loop variable, or do I need it elsewhere? Or change `AStringVector m_ModNames` to be a vector of references instead somehow? Newbie at C++...